src/components/QA/QA.test.jsx

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import { render } from '@testing-library/react';
+import QA from './QA';
+
+describe('QA component', function() {
+  it('should have a h3 displaying Questions and Answers', function() {
+    let { getByText } = render(<QA productId={65631}/>);
+    expect(getByText('Questions and Answers')).toMatchInlineSnapshot(`
```

Comment:
cool. try using getByRole query method to get the hang of testing-library

Reply:
Ill try that!